// @nestjs/common/enums/route-paramtypes.enum.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.RouteParamtypes = void 0;
var RouteParamtypes;
(function (RouteParamtypes) {
    RouteParamtypes[RouteParamtypes["REQUEST"] = 0] = "REQUEST";
    RouteParamtypes[RouteParamtypes["RESPONSE"] = 1] = "RESPONSE";
    RouteParamtypes[RouteParamtypes["NEXT"] = 2] = "NEXT";
    RouteParamtypes[RouteParamtypes["BODY"] = 3] = "BODY";
    RouteParamtypes[RouteParamtypes["RAW_BODY"] = 4] = "RAW_BODY";
    RouteParamtypes[RouteParamtypes["QUERY"] = 5] = "QUERY";
    RouteParamtypes[RouteParamtypes["PARAM"] = 6] = "PARAM";
    RouteParamtypes[RouteParamtypes["HEADERS"] = 7] = "HEADERS";
    RouteParamtypes[RouteParamtypes["SESSION"] = 8] = "SESSION";
    RouteParamtypes[RouteParamtypes["FILE"] = 9] = "FILE";
    RouteParamtypes[RouteParamtypes["FILES"] = 10] = "FILES";
    RouteParamtypes[RouteParamtypes["HOST"] = 11] = "HOST";
    RouteParamtypes[RouteParamtypes["IP"] = 12] = "IP";
})(RouteParamtypes || (exports.RouteParamtypes = RouteParamtypes = {}));
